src/decode-gre.h

```diff
@@ -74,10 +74,8 @@ typedef struct GRESreHdr_
 #define GRE_FLAG_ISSET_SSR(r)       (r->flags & 0x08)
 #define GRE_FLAG_ISSET_RECUR(r)     (r->flags & 0x07)
 #define GRE_GET_VERSION(r)   (r->version & 0x07)
-#define GRE_GET_FLAGS(r)     (r->version & 0xF8)
```

Comment:
lets keep this kind of define

Reply:
Ok cf `git checkout master -- src/decode-*.h`